components/eam/cime_config/config_component.xml

```diff
@@ -140,7 +138,7 @@
       <!-- Radiative-Convective Equilibrium (RCE) -->
       <value compset="_EAM%RCE"      >RCEMIP_EAMv1</value>
       <!-- MMF / Super-Parameterization -->
-      <value compset="2010(?:SOI)?_EAM%MMF[1XO]">2010_mmf_1mom</value>
+      <value compset="2010(?:SOI)?_EAM%MMF1"    >2010_mmf_1mom</value>
       <value compset="2010(?:SOI)?_EAM%MMF2"    >2010_mmf_2mom</value>
       <value compset="_EAM%MMF.*-RCE"           >RCEMIP_EAMv1</value>
       <value compset="_EAM%MMF.*-AQUA"          >aquaplanet_MMF-1mom</value>
```

Comment:
_EAM%MMF.*- could be replaced by _EAM%MMF1* until the MMF2 compsets are added.

Reply:
yea, but I'd like to leave it this way to minimize the changes that will come later.

Reply:
ok.